### PR TITLE
Update German translations

### PR DESCRIPTION
**Pull request by surfdude29**
Following the same process as #2917 (thanks [DeepL](https://www.deepl.com/translator)! 🙏) I have updated the German translations in this PR. I also made a few tweaks to existing translations, but of course let me know if I got something wrong there.

Unfortunately, there isn't a way that I know of to ask DeepL to produce translations with the informal tone that we settled on previously, rather than the formal tone, so those translations will definitely need correcting I'm afraid.

All corrections from native German speakers are welcome, especially previous translators including @PythooonUser, @cdfzo and @imbstt.

Thanks! :)

### CODE REVIEW

**Comment by PythooonUser** [comment]
I will have a look later!

File: src/locale/locales/de/messages.po [hunk 1]
```diff
@@ -110,7 +102,7 @@ msgstr "Konto blockiert"
 
 #: src/view/com/profile/ProfileMenu.tsx:153
 msgid "Account followed"
-msgstr ""
+msgstr "Konto folgt"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:105]: Not sure about the context here? Is it in the sense "you followed this account"?

File: src/locale/locales/de/messages.po [hunk 2]
```diff
@@ -671,11 +608,11 @@ msgstr "Suche abbrechen"
 
 #: src/view/com/modals/LinkWarning.tsx:106
 msgid "Cancels opening the linked website"
-msgstr ""
+msgstr "Abbruch des Öffnens der verlinkten Website"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:611]:
```suggestion
msgstr "Bricht das Öffnen der verlinkten Website ab"
```

File: src/locale/locales/de/messages.po [hunk 3]
```diff
@@ -1020,7 +927,7 @@ msgstr "Inhaltswarnungen"
 
 #: src/components/Menu/index.web.tsx:84
 msgid "Context menu backdrop, click to close the menu."
-msgstr "Hintergrund des Kontextmenüs, klicken, um das Menü zu schließen"
+msgstr "Hintergrund des Kontextmenüs, klicken, um das Menü zu schließen."
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:930]: Maybe separate that more?

```suggestion
msgstr "Hintergrund des Kontextmenüs; klicken, um das Menü zu schließen."
```

File: src/locale/locales/de/messages.po [hunk 4]
```diff
@@ -1087,12 +994,12 @@ msgstr "Kopieren"
 
 #: src/view/com/modals/ChangeHandle.tsx:480
 msgid "Copy {0}"
-msgstr "{} kopieren"
+msgstr "{0} kopieren"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:997]:
```suggestion
msgstr "Kopiere {0}"
```

File: src/locale/locales/de/messages.po [hunk 5]
```diff
@@ -1087,12 +994,12 @@ msgstr "Kopieren"
...
 
 #: src/components/dialogs/Embed.tsx:120
 #: src/components/dialogs/Embed.tsx:139
 msgid "Copy code"
-msgstr ""
+msgstr "Kopieren diesen Code"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:1002]:
```suggestion
msgstr "Kopiere den Code"
```

File: src/locale/locales/de/messages.po [hunk 6]
```diff
@@ -1141,7 +1044,7 @@ msgstr "Konto erstellen"
 #: src/components/dialogs/Signin.tsx:86
 #: src/components/dialogs/Signin.tsx:88
 msgid "Create an account"
-msgstr ""
+msgstr "Ein Konto erstellen"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:1047]:
```suggestion
msgstr "Erstelle ein Konto"
```

File: src/locale/locales/de/messages.po [hunk 7]
```diff
@@ -1562,17 +1433,17 @@ msgstr "E-Mail:"
 
 #: src/components/dialogs/Embed.tsx:112
 msgid "Embed HTML code"
-msgstr ""
+msgstr "HTML-Code zum Einbetten"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:1436]:
```suggestion
msgstr "Bette HTML-Code ein"
```

File: src/locale/locales/de/messages.po [hunk 8]
```diff
@@ -2373,35 +2207,35 @@ msgstr "Journalismus"
 
 #: src/components/moderation/LabelsOnMe.tsx:59
 msgid "label has been placed on this {labelTarget}"
-msgstr ""
+msgstr "Etikett wurde auf diesem {labelTarget} platziert"
```
Comment (PythooonUser) [inline src/locale/locales/de/messages.po:2210]: I think we can keep `label` as `Label` everywhere.

File: src/locale/locales/de/messages.po (hunk 1 above)
Comment (surfdude29) [reply src/locale/locales/de/messages.po:105]: Yes this is the toast (pop-up notification) that is shown when you click/tap Follow on someone's profile. So "you followed this account" or alternatively "this account has been followed by you".

File: src/locale/locales/de/messages.po (hunk 8 above)
Comment (surfdude29) [reply src/locale/locales/de/messages.po:2210]: OK I'll try and do some changes to all the strings affected.

**Review by surfdude29 — COMMENTED** [review]
Thanks once more @PythooonUser! :)

I've applied all your suggested changes and I'll try and change some of the strings involving `label`.

**Comment by cdfzo** [comment]
@surfdude29 i'll have a look today :) could you please rebase?

**Comment by surfdude29** [comment]
> @surfdude29 i'll have a look today :) could you please rebase?

That was good timing! 😄 When I saw your email I was actually in the middle of working on an update (I started from scratch as I thought that was easiest).

The new PR is #4742 so I'll close this one.

**Pull request closed without merging**